### ISSUE TITLE
The alt text reads Dice instead of the dice face... 

### ISSUE THREAD

**Issue by shubhraagarwal** [gouri-panda/Pig-game#10]
The default alt text has been set to "Dice", it should be showing the dice face, because even if the image dosen't load, people will still be able to see the die they rolled and it will also help people using screen readers.

**Comment by shubhraagarwal** [comment]
I went ahead and solved this issue. I have created a pull request for the same.
